src/renderer/components/Dex.vue

```diff
@@ -85,9 +80,12 @@ export default {
 
   created() {
     this.setConnected();
-    this.interval = setInterval(() => {
+    this.connectionStatusInterval = setInterval(() => {
       this.setConnected();
     }, 1000);
+    this.marketsRefreshInterval = setInterval(() => {
+      this.loadMarkets();
+    }, this.$constants.intervals.MARKETS_POLLING);
```

Comment:
I wonder if these intervals should have any jitter in them in the case we ever had an event where a bunch of people started their wallets at right at the same time. In that case, a vast majority of the calls would be falling in the same minute out of 5 minutes rather than spread evenly across the polling interval.